diff -Nur a/alevt-cap.c b/alevt-cap.c
--- a/alevt-cap.c	2025-09-06 19:41:50.000000000 +0200
+++ b/alevt-cap.c	2026-08-02 15:16:21.277261789 +0200
@@ -4,6 +4,7 @@
 #include <signal.h>
 #include <unistd.h>
 #include <locale.h>
+#include <sys/stat.h>
 
 #include "vt.h"
 #include "misc.h"
@@ -219,6 +220,10 @@
     char *progname = NULL;
     int txtpid = -1;
 
+    struct stat statbuf;
+    if (stat(vbi_name, &statbuf) == -1)
+        vbi_name = "/dev/v4l/vbi0";
+
     setlocale(LC_ALL, "");
     setprgname(argv[0]);
 
diff -Nur a/alevt-date.c b/alevt-date.c
--- a/alevt-date.c	2026-08-02 15:16:21.235976119 +0200
+++ b/alevt-date.c	2026-08-02 15:16:21.277330599 +0200
@@ -6,6 +6,7 @@
 #include <unistd.h>
 #include <signal.h>
 #include <time.h>
+#include <sys/stat.h>
 #include <locale.h>
 #include <langinfo.h>
 
@@ -179,6 +180,10 @@
     char *progname = NULL;
     int txtpid = -1;
 
+    struct stat statbuf;
+    if (stat(vbi_name, &statbuf) == -1)
+        vbi_name = "/dev/v4l/vbi0";
+
     setlocale(LC_ALL, "");
     setprgname(argv[0]);
 
diff -Nur a/main.c b/main.c
--- a/main.c	2025-09-06 19:41:50.000000000 +0200
+++ b/main.c	2026-08-02 15:16:21.277380190 +0200
@@ -2,6 +2,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <locale.h>
+#include <sys/stat.h>
 
 #include "vt.h"
 #include "misc.h"
@@ -199,6 +200,10 @@
     int opt, ind;
     char *arg;
 
+    struct stat statbuf;
+    if (stat(vbi_name, &statbuf) == -1)
+      vbi_name = "/dev/v4l/vbi0";
+
     setlocale(LC_ALL, "");
     setprgname(argv[0]);
 
